Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml

```
@@ -1,6 +1,8 @@
 - type: vendingMachineInventory
   id: SecDrobeInventory
   startingInventory:
+    ClothingOuterSecLeather: 2
+    ClothingOuterSecGreenAlt: 2
```

```suggestion
    ClothingOuterSecLeather: 2 # DS14-Soyuz
    ClothingOuterSecGreenAlt: 2 # DS14-Soyuz
```
А так же перенести в конец списка